public/apps/configuration/utils/tenant-utils.tsx

```diff
@@ -36,9 +38,15 @@ import { httpDelete, httpGet, httpPost } from './request-utils';
 import { getResourceUrl } from './resource-utils';
 
 export const globalTenantName = 'global_tenant';
+export const GLOBAL_TENANT = '';
+export const PRIVATE_TENANT = '__user__';
```

Comment:
There are constants defined for the same [here](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/server/multitenancy/tenant_resolver.ts#L21)
We can either reuse them or use these in that file to avoid duplicity.

Reply:
done